zodburi/__init__.py

```diff
@@ -16,19 +17,41 @@ def resolve_uri(uri):
     else:
         raise KeyError('No resolver found for uri: %s' % uri)
 
+connection_parameters = '''
+  pool_size pool_timeout cache_size cache_size_bytes
+  historical_pool_size historical_cache_size historical_cache_size_bytes
+  historical_timeout large_record_size
+  '''.strip().split()
```

Comment:
Good point.

Shall I also add documentation to that documentation? :)  The existing documentation only documents the type and default.